templates/katello.yaml.erb

```diff
@@ -18,14 +18,15 @@
     :oauth_secret: <%= scope['katello_devel::oauth_secret'] %>
     :ca_cert_file: <%= scope['katello_devel::candlepin_ca_cert'] %>
 
+  :candlepin_events:
+    :ssl_cert_file: <%= scope['certs::candlepin::client_cert'] %>
+    :ssl_key_file: <%= scope['certs::candlepin::client_key'] %>
+    :ssl_ca_file: <%= scope['katello_devel::candlepin_ca_cert'] %>
```

Comment:
I think we should introduce `$certs::candlepin::ca_cert` for consistency. Will see if I can come up with a PR.

Reply:
Looks like this variable is already available, but it's a different file.

IMHO we should introduce a `certs::candlepin_client` class and katello should use that.